pkgs/development/python-modules/plotly/default.nix

```diff
@@ -116,6 +120,9 @@ buildPythonPackage rec {
     "test_dependencies_not_imported"
     # FAILED test_init/test_lazy_imports.py::test_lazy_imports - AssertionError: assert 'plotly' not in {'IPython': <module 'IPython' from '...
     "test_lazy_imports"
+    # requires vaex and polars, vaex is not packaged
+    "test_build_df_from_vaex_and_polars"
+    "test_build_df_with_hover_data_from_vaex_and_polars"
```

Comment:
```suggestion
    "test_build_df_with_hover_data_from_vaex_and_polars"
    # Cannot launch kaleido as a subprocess. This would need a passthru test.
    "test_kaleido_engine_to_image_returns_bytes"
    "test_kaleido_fulljson"
    "test_bytesio"
    "test_png_renderer_mimetype"
    "test_svg_renderer_show"
    "test_pdf_renderer_show_override"
    "test_mimetype_combination"
    # Cannot run on systes without orca
    "test_scraper"
    "test_bytesio"
```

Reply:
How about

```suggestion
    "test_build_df_with_hover_data_from_vaex_and_polars"
] ++ lib.optionals stdenv.hostPlatform.isDarwin [
    # Cannot launch kaleido as a subprocess. This would need a passthru test.
    "test_kaleido_engine_to_image_returns_bytes"
    "test_kaleido_fulljson"
    "test_bytesio"
    "test_png_renderer_mimetype"
    "test_svg_renderer_show"
    "test_pdf_renderer_show_override"
    "test_mimetype_combination"
    # Cannot run on systes without orca
    "test_scraper"
    "test_bytesio"
];
```

Reply:
@Pandapip1 that's even better.